yarn.lock

```diff
@@ -1251,7 +1251,7 @@
     jest-runtime "^24.9.0"
     jest-util "^24.9.0"
     jest-worker "^24.6.0"
-    node-notifier "^5.4.2"
+    node-notifier "^8.0.1"
```

Comment:
so this a dependency of `@jest/reporters` version 24.9.0. did you try updating that dependency?